### PR TITLE
remove viewlet

### PR DESCRIPTION
**Pull request by erral**
See https://github.com/plone/plone.app.layout/issues/286

### CODE REVIEW

**Comment by Rudd-O** [comment]
https://github.com/plone/plone.app.layout/issues/286#issuecomment-1027454879

File: Products/CMFPlone/profiles/default/viewlets.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <object>
   <order manager="plone.portaltop" skinname="Plone Default">
-    <viewlet name="plone.header" />
```
Comment (ale-rt) [inline Products/CMFPlone/profiles/default/viewlets.xml:4]: This requires an upgrade step, anyway right?
In addition I do not see any point in having an empty:
```
<order manager="plone.portaltop" skinname="Plone Default">
</order>
```

Comment (erral) [reply Products/CMFPlone/profiles/default/viewlets.xml:4]: right: https://github.com/plone/plone.app.upgrade/pull/268/files

Yeah... I wasn't sure if we need the order manager empty in order to configure it properly or not... I have never been a big fan of viewlets.xml and never could get it to work properly

**Review by MrTango — APPROVED** [review]
LGTM

File: Products/CMFPlone/profiles/default/viewlets.xml (hunk 1 above)
Comment (mauritsvanrees) [reply Products/CMFPlone/profiles/default/viewlets.xml:4]: If a viewlet is no longer registered for a manager in zcml, then it should not be needed to remove it in xml. But does not hurt. I just tried.

**Review by mauritsvanrees — APPROVED** [review]

**Pull request merged**